packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -2,11 +2,10 @@ exports[`test should render a date picker with disabled dates 1`] = `
 <ResponsiveElement
   className="terra-DatePicker"
   defaultElement={
-    <DatePicker
+    <t
```

Comment:
https://github.com/Hacker0x01/react-datepicker/compare/v0.44.0...v0.46.0#diff-f662ae56197d5c888897a02b982f0ad3L18

I think this change is because they removed the `displayName` from the datepicker component